src/FormCollection/Resources/views/form_theme_div.html.twig

```diff
@@ -0,0 +1,51 @@
+{%- block button_add -%}
+    {%- set attrDataAction = (attr['data-action']|default('') ~ ' ' ~ controllerName ~ '#add')|trim -%}
+    <button data-action="{{ attrDataAction }}"
+            class="{{ button_add_options.class }}"  type="button">{{ button_add_options.label|trans }}</button>
+{%- endblock button_add -%}
+
+{%- block button_delete -%}
+    {%- set attrDataAction = (attr['data-action']|default('') ~ ' ' ~ controllerName ~ '#delete')|trim -%}
+    <button data-action="{{ attrDataAction }}"
+            class="{{ button_delete_options.class }}" type="button">{{ button_delete_options.label|trans }}</button>
+{%- endblock button_delete -%}
+
+{% block ux_collection_widget -%}
```

Comment:
I'd love to NOT need this block. Instead, inside of `UxCollectionType`, in `finishView()`, we should (I *think*) be able to add all of the attributes we need. For example, adding a `data-controller` attribute should be easy enough... as well as all of the Stimulus "values".

Even the `attrDataTarget`, I believe, could be added inside `finishView`. There, you're passed a `FormView` object, you should be able to iterate over `$formView->children` to then add the `attrDataTarget` to each (and you would do the same with `$formView->vars['prototype']`.

I'm pushing for this because, if we accomplish this, then if the user has already customized their `ux_collection_widget` block in a form theme, then we are not overriding their work. Also, it would allow me to, for example, do something like this:

```twig
<div
    {{ stimulus_controller('symfony/ux-form-collection/collection') }}
    data-prototype="{{ form_row(form.files.vars.prototype)|e('html_attr') }}"
>
    Files:
    <button
        class="btn btn-sm btn-secondary"
        {{ stimulus_action('symfony/ux-form-collection/collection', 'add') }}
    >+</button>
    {% for fileForm in form.files %}
        {{ form_row(fileForm) }}
    {% endfor %}
</div>
```

Currently that doesn't work because I have *avoided* rendering the `ux_collection_widget`, and so the children (and prototype) are all missing their targets.